### PR TITLE
SOC-1591 | Display line breaks in posts and replies

### PR DESCRIPTION
**Pull request by Bloodmoon24**
https://wikia-inc.atlassian.net/browse/SOC-1591
@kvas-damian @rafalkalinski  please CR

### CODE REVIEW

File: front/scripts/main/mixins/discussion-parsed-content.js [hunk 1]
```diff
@@ -5,13 +5,13 @@ import App from '../app';
...
 	/**
-	 * Returns content with links created from urls
+	 * Returns content with links created from urls and converts \n to <br>
 	 * @returns {string}
 	 */
 	parsedContent: Ember.computed(function () {
 		const escapedContent = Ember.Handlebars.Utils.escapeExpression(
 			this.get('post.rawContent')
-		);
+		).replace(/(?:\r\n|\r|\n)/g, '<br>');
```
Comment (rafalkalinski) [inline front/scripts/main/mixins/discussion-parsed-content.js]: Did you consider implementing it in a way, that we put 2 br at max. It means, that one \r or \n gives us one \<br\>, two give us 2 x \<br\>, but more linebreaks (or lineends) will always generate two \<br\>s

Comment (rogatty) [inline front/scripts/main/mixins/discussion-parsed-content.js]: Please extract to front/scripts/mercury/utils and write some tests for it.

Comment (Bloodmoon24) [inline front/scripts/main/mixins/discussion-parsed-content.js]: @rafalkalinski I asked designer about this and he said that there should be no limitations, only trimming at the beginning and the ending of the string.
@rogatty I extracted it and wrote tests.

File: front/scripts/mercury/utils/parseNewLine.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+/**
+ * @param {string} rawContent
+ * @returns {string}
+ */
+export function parseNewLine(rawContent) {
```
Comment (kvas-damian) [inline front/scripts/mercury/utils/parseNewLine.js]: While creating utils please make sure they're reusable and have right names.

In this case i believe ```nl2br``` is well known name. In addition we can move ```trim()``` out of this method.

**Comment by kvas-damian** [comment]
Thanks for unit tests! :+1:

File: front/scripts/mercury/utils/parseNewLine.js (hunk 2 above)
Comment (Bloodmoon24) [inline front/scripts/mercury/utils/parseNewLine.js]: Done

**Comment by kvas-damian** [comment]
LGTM

File: front/scripts/mercury/utils/nl2br.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+/** 
+ * This function is parsing \n, \rn, \r to <br>
+ * @param {string} rawContent
+ * @returns {string}
+ */
+export function nl2br(rawContent) {
```
Comment (rogatty) [inline front/scripts/mercury/utils/nl2br.js]: How about `export default`?

**Comment by bkoval** [comment]
LGTM

**Comment by kvas-damian** [comment]
LGTM

**Comment by rogatty** [comment]
LGTM

**Comment by rafalkalinski** [comment]
LGTM

**Pull request merged**